float64 radar1_dx_
float64 radar1_dy_
float64 radar1_vx_
float64 radar1_vy_